vault/auth.go

```diff
@@ -189,7 +189,7 @@ func (c *Core) disableCredential(ctx context.Context, path string) error {
 
 	if backend != nil {
 		// Revoke credentials from this path
-		if err := c.expiration.RevokePrefix(fullPath, true); err != nil {
+		if err := c.expiration.RevokePrefix(ctx, fullPath, true); err != nil {
```

Comment:
I wonder if we should use the active context here? This could potentially take a long time to run

Reply:
We do but I am wondering where the best place to set the value to the active context, when it is called or the top level function.  I opted for the top level function but am not sure if that is right.

See https://github.com/hashicorp/vault/pull/5021/files#diff-6064434ffbb0a34a01f295d95a129559R608.